docs/docs/python/integrations/pyoxidizer.mdx

```diff
@@ -7,6 +7,14 @@ Creating Python binaries through PyOxidizer.
 
 ---
 
+:::caution (March 17, 2024) PyOxidizer is not actively maintained
+As mentioned in [PyOxidizer issue #741](https://github.com/indygreg/PyOxidizer/issues/741) by indygreg, development on PyOxidizer is effectively stagnant. Please take this into consideration before using PyOxidizer in a new project, as this may affect PyOxidizer support within Pants as well.
+
+> tl;dr shifting personal priorities have resulted in me de-prioritizing PyOxidizer (and other open source projects). At this point in time, the future of PyOxidizer is uncertain, possibly dead.
+
+Consider using [Scies](https://github.com/a-scie/jump/blob/main/docs/packaging.md) as an alternative for creating portable Python distributions. This is how Pants itself [is distributed](https://github.com/pantsbuild/scie-pants).
```

Comment:
It seems a bit... mean to be suggesting scies when there's not currently an easy way to construct them in Pants. Do we have any hint of a scie backend yet?

Reply:
I was on the fence about the alternative recommendation - but it felt kinda ... I dunno, weird, to suggest not using a deprecated package but then being like "alright, carry on" when we have an alternative that we use ourselves.

I have a limited `scie` plugin here, which I currently use on a handful of projects: https://github.com/sureshjoshi/pants-plugins/tree/main/pants-plugins/experimental/scie (published as `plugins = ["robotpajamas.pants.scie==0.0.2"]`)

For example:
https://github.com/sureshjoshi/pantsible/blob/main/BUILD.pants

The primary reason why I haven't cleaned it up and mainlined it, is that when I was originally writing it a year ago - John mentioned a plan was to maybe build this sort of [capability into pex (long term)](https://pantsbuild.slack.com/archives/G0107QFGGAW/p1674519720278109?thread_ts=1674517617.545449&cid=G0107QFGGAW). So, didn't seem right to build a mainline plugin that we were planning to deprecate. 🤷🏽 

If there's interest, I can strip this paragraph out, mainline the plugin, and then add the paragraph back in? @huonw

Reply:
Cool, great to see a plugin.

I'm inclined to do your proposed plan: Given these docs are just in the context of using PyOxidizer within Pants, I think it's fine for it to be informational (i.e. "if you're planning to use PyOxidizer, here's a not-so-fun fact you might want to know"). If/when we get a better alternative, we can add it in.

Also, it's a bit pedantic and may disappear soon, but describing pants as distributed as a scie is also a bit misleading: Pants itself (i.e. the code in this repo) is distributed as a PEX, with scie-pants being a scie that downloads that PEX. This repository "barely" knows about scie at all (basically just some code in `pants_runner.py` that validates versions by checking env vars scie-pants sets).